guides/source/active_record_callbacks.md

```diff
@@ -178,8 +178,8 @@ order in which they will get called** during the respective operations:
 * [`after_save`][]
 * [`after_commit`][] / [`after_rollback`][]
 
-`after_commit` / `after_rollback` examples can be found
-[here](active_record_callbacks.html#after-commit-and-after-rollback).
+Find [examples for using `after_commit` /
```

Comment:
```suggestion
Jump to the [`after_commit` /
`after_rollback` section below](active_record_callbacks.html#after-commit-and-after-rollback) for examples using these two callbacks.
```

Reply:
I think I'm going to leave out the "below" (and next) if you're ok with that?